Sources/Decorator/Extensions+HKElectrocardiogram.swift

```diff
@@ -88,6 +108,18 @@ extension HKElectrocardiogram.Classification: CustomStringConvertible {
 // MARK: - CustomStringConvertible
 @available(iOS 14.0, *)
 extension HKElectrocardiogram.SymptomsStatus: CustomStringConvertible {
+    public var key: String {
```

Comment:
the same as in the [comment](https://github.com/VictorKachalov/HealthKitReporter/pull/16/files#r912658773)